manifests/vanilla/csi-snapshot-validatingwebhook.yaml

```diff
@@ -84,7 +84,7 @@ spec:
       serviceAccountName: snapshot-webhook
       containers:
         - name: snapshot-validation
-          image: registry.k8s.io/sig-storage/snapshot-validation-webhook:v6.3.3 # change the image if you wish to use your own custom validation server image
+          image: registry.k8s.io/sig-storage/snapshot-validation-webhook:v7.0.0 # change the image if you wish to use your own custom validation server image
```

Comment:
Change you change to 7.0.1?
https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v7.0.1

Reply:
Done